### PR TITLE
AMIGAOS: Automate install process

### PR DESCRIPTION
**Pull request by raziel-**
Could you please rename the icon files in gui/media to
scummvm-tools.info
and
scummvm-tools_drawer.info
?

Thanks a lot

### CODE REVIEW

**Comment by sev-** [comment]
Thanks!

**Comment by raziel-** [comment]
@sev-

The files still need to be renamed, please

**Comment by sev-** [comment]
I just renamed the files, however, it would be helpful to provide me with the exact commands to avoid any mistakes. Like:
```
mv ScummVM-Tools.info scummvm-tools.info
mv scummvm-tools.drawer.info scummvm-tools_drawer.info
```

**Comment by raziel-** [comment]
@sev-

Sorry for all the extra work I'm causing.
I dont even know what "mv" is or does, I assume it rename files  :-)

Thanjs a lot, you are gold

File: Makefile [hunk 1]
```diff
@@ -223,41 +223,51 @@ osxsnap: bundle scummvm-tools-cli-static
...
-	mkdir -p $(AMIGAOS4PATH)
-	mkdir -p $(AMIGAOS4PATH)/graphics
-	mkdir -p $(AMIGAOS4PATH)/tools
-	mkdir -p $(AMIGAOS4PATH)/tools/media
-	cp $(srcdir)/gui/media/detaillogo.jpg $(AMIGAOS4PATH)/tools/media/
-	cp $(srcdir)/gui/media/logo.jpg $(AMIGAOS4PATH)/tools/media/
-	cp $(srcdir)/gui/media/tile.gif $(AMIGAOS4PATH)/tools/media/
-	$(STRIP) construct_mohawk$(EXEEXT) -o $(AMIGAOS4PATH)/tools/construct_mohawk$(EXEEXT)
+# Special target to create an AmigaOS snapshot installation.
+amigaosdist: all
```
Comment (sev-) [inline Makefile:227]: You changed the target from `amigaos4dist` to `amigaosdist` here. Is this intended? If it is so, then I'll adjust the buildbot. If it is not, I'd then revert the target name change.

Comment (raziel-) [reply Makefile:227]: It was intended.
I'm syncronising towards scummvm main where the target is the same.

Thank you

P.s. small adaptations to the tools coming up

**Pull request merged**